dev/drivers/scripts/subseasonal/stats/jevs_subseasonal_cfs_grid2grid_stats.sh

```diff
@@ -29,7 +29,6 @@ source $HOMEevs/modulefiles/subseasonal/subseasonal_stats.sh
 export MET_ROOT=/apps/ops/para/libs/intel/${intel_ver}/met/${met_ver}
 export MET_BASE=${MET_ROOT}/share/met
 export PATH=${MET_ROOT}/bin:${PATH}
```

Comment:
I think that these three lines should probably deleted.  The modulefile takes care of bringing in the current version of MET, and plus you're pointing to para here.  Are you using MET_ROOT and MET_BASE throughout your code?

Reply:
I do use MET_ROOT and METPLUS_PATH as environ variables throughout all my ush scripts. I'm not sure why it still says para, but I can change it to the prod path.

Reply:
@ShannonShields-NOAA Nothing in EVS should says para for the ops code delivery, so change this to prod, keep it all for now, and we'll worry about it later as a  bugfix or EE2 fix. 

Based on the [loading instructions for MET-11.0.2 in /prod](https://dtcenter.org/community-code/metplus/metplus-5-0-existing-builds), this line might be slightly changed to:
module use /apps/prod/lmodules/intel/19.1.3.304

Reply:
Also, the parm/machine.conf uses MET_INSTALL_DIR = {ENV[MET_ROOT]}. So I think the scripts will fail if these lines are removed.

Reply:
I think that MET_ROOT is included as part of the met library install.  I do not have MET_ROOT defined in any of my components and MET still works.  

In the working test below (with the COMIN fix), I actually commented out these lines:

```
#export MET_ROOT=/apps/ops/para/libs/intel/${intel_ver}/met/${met_ver}
#export MET_BASE=${MET_ROOT}/share/met
#export PATH=${MET_ROOT}/bin:${PATH}
```

So I feel you can safely remove all of these lines.

Perry